TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -0,0 +1,21 @@
+@objc public class TradeItSDK: NSObject {
+    private static var apiKey: String?
+    private static var environment: TradeitEmsEnvironments?
+
+    public static let launcher = TradeItLauncher()
+    private (set) public static var linkedBrokerManager: TradeItLinkedBrokerManager!
+    private (set) public static var marketDataService: TradeItMarketService!
+    private (set) public static var brokerCenterService: TradeItBrokerCenterService!
```

Comment:
@mitochondrion any ideas for how to warn if they haven't called `configure` before? Computed properties won't allow you to throw an exception or anything like that.